dashboard/app/controllers/api_controller.rb

```diff
@@ -204,7 +204,7 @@ def user_progress_for_stage
       slog(
         tag: 'activity_start',
         script_level_id: script_level.try(:id),
-        level_id: level.id,
+        level_id: level.contained_levels.empty? ? level.id : level.contained_levels.first.id,
```

Comment:
We don't actually render more than one contained level anywhere. There's one level that has multiple contained levels, but it only shows the first and isn't actually used in a script: https://levelbuilder-studio.code.org/levels/4562